bedrock/security/management/commands/update_security_advisories.py

```diff
@@ -208,9 +234,8 @@ def printout(msg, ending=None):
 
         errors = []
         updates = 0
-        all_files = get_all_mfsa_files()
+        all_files = get_all_file_names()
         for mf in all_files:
-            mf = os.path.join(ADVISORIES_PATH, mf)
```

Comment:
Basically yes, it was never needed. It was working because os.path.join does some fancy things to resolve paths, but it wasn't required and was confusing me as to why it was there. I think this is more clear.

Reply:
👍